demo/app/Sharp/Posts/Commands/ExportPostsToCsvCommand.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
No to the whole command: duplicate + we have a "wizard" command coming in PR https://github.com/code16/sharp/pull/401